openapi.yml

```diff
@@ -690,3 +736,19 @@ components:
           $ref: '#/components/schemas/Interface'
         endpoint_b:
           $ref: '#/components/schemas/Interface'
+    Tag_ranges: # Can be referenced via '#/components/schemas/Tag_ranges'
+      type: object
+      required:
+        - tag_type
+        - tag_ranges
+      properties:
+        tag_type:
+          type: string
```

Comment:
The tag_type as a `str` as we are discussing on https://github.com/kytos-ng/kytos/pull/409, it should've been a str only if we were to evolve `TAGType` enum as str too, which we won't not to break mef_eline, let's keep as int here since it's been historically this way.

Reply:
Also we since we'll only support `TAGType.VLAN.value = 1` for now, let's also restrict this as a enum in the API. One day, if we support other ones, it's just a matter of allowing here too.

Reply:
Changed type to integer.